lib/config.js

```diff
@@ -9,6 +9,7 @@ var autoconfig = function(overrides) {
     dataTopicPrefix: ':cloud:data',
     persistentStore: process.env.WFM_USE_MEMORY_STORE !== "true",
     syncOptions: {
+      "syncFrequency": 5,
```

Comment:
@wtrocki A short section about the sync frequencies has been added to the `raincatcher-sync` PR as that's where these values are used, and a comment has been added here to `config.js`. Would you mind taking a look?